tenders-finder/.env

```
@@ -0,0 +1,3 @@
+VITE_SUPABASE_PROJECT_ID="dksfgbuuciwhicmpdkys"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)